src/api/urls.py

```diff
@@ -1,5 +1,6 @@
 from django.urls import include, path
 from rest_framework.routers import DefaultRouter
+from .views import readiness_check, liveness_check
 
 from api import views
```

Comment:
The import statement `from api import views` is redundant since specific views are already imported in line 3. You can remove this line to clean up the code.